bin/server.js

```diff
@@ -9,9 +9,10 @@ var AwsManager = require('../provisioner/aws-manager');
 var v1 = require('../routes/v1');
 var Aws = require('multi-region-promised-aws');
 var _ = require('lodash');
+var series = require('../provisioner/influx-series');
 
 /** Launch server */
-var launch = function (profile) {
+var launch = function (profile, useNullInfluxDrain) {
```

Comment:
Why not use `config.get('influx') === null` instead, and then disable influxdb in `config/test.js`.